ddapm_test_agent/agent.py

```diff
@@ -227,6 +227,7 @@ async def handle_info(self, request: Request) -> web.Response:
                 ],
                 "feature_flags": [],
                 "config": {},
+                "client_drop_p0s": True,
```

Comment:
should this response value always be semi-fixed based on recent version of the agent?

should we have tests that validate we return the same value as the agent?

Reply:
AFAICT it's just `true` if the stats endpoint is implemented... idk why there's a separate option for it if that is the case though 🤔